src/main/java/qna/domain/Answer.java

```diff
@@ -32,6 +38,10 @@ public Answer(Long id, User writer, Question question, String contents) {
         this.contents = contents;
     }
 
+    protected Answer() {
```

Comment:
엔티티를 위한 성생자 👍 
```suggestion
    protected Answer() {}
```
[컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 줘도 좋을 것 같아요...!